web/js/components/layer/product-picker/browse/measurement-layer-row.js

```diff
@@ -12,105 +12,104 @@ import {
 } from '../../../../modules/layers/actions';
 import SelectedDate from '../../../selected-date';
 import getSelectedDate from '../../../../modules/date/selectors';
+import { getLayerNoticesForLayer } from '../../../../modules/notifications/util';
 
 /*
  * A scrollable list of layers
  * @class LayerList
  * @extends React.Component
  */
-class MeasurementLayerRow extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      tooltipOpen: false,
-    };
-    this.onClick = this.onClick.bind(this);
-    this.toggleTooltip = this.toggleTooltip.bind(this);
-  }
+function MeasurementLayerRow (props) {
+  const {
+    isEnabled,
+    isMobile,
+    removeLayer,
+    addLayer,
+    layer,
+    measurementId,
+    title,
+    selectedDate,
+    layerNotices,
+  } = props;
+  const layerIsAvailable = available(layer.id, selectedDate, [layer]);
```

Comment:
I think this is fine here since I'm always using the inverse but, generally I'm not a fan of boolean var names that have negatives in them if you ever need to check the inverse.  For example checking `!layerIsNotAvailable` becomes less readable I think.